src/grisp_cryptoauth_cert.erl

```diff
@@ -10,6 +10,7 @@
          compress/3,
          decompress/2,
          print/1,
+         rdn_sequence/1,
```

Comment:
I'd prefer a `distinguished_name` method (bc it's all about building up a DN out of RDNs).